users/tests.py

```diff
@@ -450,3 +454,85 @@ def test_user_profile_delete_unauthorized(self):
         response = self.client.post(url, content_type='application/json')
 
         self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+
+class UserConfirmEmailTest(BaseSingleUserTest):
+    """Tests the confirm_email endpoint. """
+
+    def setUp(self):
+        super().setUp()
+
+        user = User.objects.get(username=self._user['username'])
+        user.person.email_confirmed = False
```

Comment:
Set `email_confirmed` to `False` only in `test_confirm_email` and `test_confirm_email_with_invalid_token` test methods.